contacts.html

```diff
@@ -0,0 +1,215 @@
+          style="background-color: white; height: 5rem;"
+        >
+          <a class="col-6 col-md-5 navbar-brand" href="#">
+            <div class="row" style="color: black;">
+              <div class="col-12 col-md-auto p-0">
+                <h5 class="h3 m-0">
+                  <i
+                    class="fa fa-square m-0"
+                    style="font-size: 1.6rem; color: blue;"
+                  ></i>
+                  MAYA NELSON
+                </h5>
+              </div>
+              <div class="col-12 col-md-auto p-0">
+                <p class="m-0">
+                  <span class="d-none d-md-inline" style="font-size: 1.4rem;">
+                    &nbsp;/
+                  </span>
+                  Project Manager
+                </p>
+              </div>
+            </div>
+          </a>
+
+          <button
+            class="navbar-toggler"
+            type="button"
+            data-bs-toggle="collapse"
+            data-bs-target="#navbarLinks"
+            aria-controls="navbarLinks"
+            aria-label="Toggle navigation"
+          >
+            <i class="fa fa-bars"></i>
+          </button>
+          <div
+            class="justify-content-end p-0 collapse navbar-collapse -success"
+            id="navbarLinks"
+          >
+            <ul class="navbar-nav align-items-end justify-content-end">
+              <li class="nav-item">
+                <a class="nav-link" href="./index.html">ABOUT ME</a>
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" href="resume.html">RESUME</a>
+              </li>
+
+              <li class="nav-item">
+                <a
+                  class="nav-link"
+                  href="projects.html"
+                  >PROJECTS</a
+                >
+              </li>
+              <li class="nav-item">
+                <a class="nav-link" 
+                style="font-weight: bold; color: blue;"
+                href="#">CONTACTS</a>
+              </li>
+            </ul>
+          </div>
+        </nav>
+      </div>
+    </header>
+
+    <main class="container-fluid" style="height: auto;">
+      <div class="main-background row" id="secondDiv"></div>
+      <div id="firstDiv" class="main-content row justify-content-center">
+        <div class="col-12 col-md-7 mb-4 bg-transparent">
+          <h5 class="h3 text-center">
+            <i
+              class="fa fa-square m-0"
+              style="font-size: 1.2rem; color: blue;"
+            ></i>
+            CONTACTS
+          </h5>
+        </div>
+        <div class="col-12 col-md-7 content-card">
+          <div class="row justify-content-between align-items-center">
+            <div class="col-12 col-md-6 p-4">
+              <label for="nameinput"
+                >First Name
+                <span style="color: blue; font-size: 1.3rem;">*</i>
+              </label>
+              <input type="text" id="nameinput" class="form-control" />

```

Comment:
Don't rely on Bootstrap for anything other than layout. Bootstrap can help with giving you a starting point for elements like form inputs or buttons, but you must still add your own CSS to re-create the design provided to you.
The form fields in the design look different, they have a bottom black border which turns bright blue on hover. Add your own custom class to these fields to re-create the design.